### ISSUE TITLE
'bool' object is not iterable

### ISSUE THREAD

**Issue by Frigyes06** [makeasnek/FindTheMag2#28]
Output
```Warning: multiple values found for enablesidestaking in gridcoin config file at D:\Gridcoin\gridcoinresearch.conf using the first one we found
Traceback (most recent call last):
  File "D:\FindTheMag2\main.py", line 2535, in <module>
    ALL_BOINC_PROJECTS=loop.run_until_complete(get_all_projects(rpc_client))
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\lib\asyncio\base_events.py", line 649, in run_until_complete
    return future.result()
  File "D:\FindTheMag2\main.py", line 1498, in get_all_projects
    for project in project_status_reply:
TypeError: 'bool' object is not iterable```

**Comment by makeasnek** [comment]
Thank you for the bug report! I'll look into this and make sure it's fixed in the new version

**Comment by makeasnek** [comment]
Fixed in latest release v 3.0
